docs/FAQ.md

```diff
@@ -1,3 +1,8 @@
+---
```

Comment:
Oh, it just occurred to me: Are now all files in `docs` pulled into the docs website? 

Previously the content here was developer documentation, i.e. docs for contributing to Cadence, not for using Cadence. Maybe we should move the content somewhere else?

Reply:
I think that would be a good idea. At some point in the future, we would have to tag the documentation with a (major/minor) version of Cadence. When we do that. having separate repos, leaves us the freedom to do individual release cycles for docs and Cadence. i.e: We can keep improving/fix (patching) documentation without doing a release on the Cadence.

Reply:
ok, I'm open to suggestions. A manual deploy of the docs site is still required fro changes from these files to show up. 
Maybe we could create a guides folder to hold the other docs that don;t need to be at the root of this folder. 

Also, I believe some of these docs may be obsolete?